MachObfuscator/main.swift

```diff
@@ -4,10 +4,11 @@ private func main() {
     let options = Options.fromCommandLine()
     guard !options.help,
         let appDirectory = options.appDirectory,
-        let mangler = SymbolManglers.mangler(byKey: options.manglerKey) else {
+        let manglerType = SymbolManglers(rawValue: options.manglerKey) else {
         return
     }
     LOGGER = SoutLogger(options: options)
+    let mangler = manglerType.resolveMangler(machOViewDoomEnabled: options.machOViewDoom)
     let obfuscator = Obfuscator(directoryURL: appDirectory, mangler: mangler)
     print(Options.usage)
```

Comment:
Damn :/ Previous merge gone wrong on github :/ This line should be in `guard ... else {}` clause.